tests/Backend/Mixed/SharingTest.php

```diff
@@ -1162,6 +1162,148 @@ function (array $column) {
         $this->assertCount(5, $workspaceTableData);
     }
 
+    /**
+     * @dataProvider sharingBackendData
+     * @throws ClientException
+     */
+    public function testShareBucketToProject($backend)
+    {
+        $this->initTestBuckets($backend);
```

Comment:
Ohladne tej init metody tak to poriesime v PR potom kde rozdelujem aktualne endopointy a uz si tam ktomu nieco pisal https://github.com/keboola/storage-api-php-client/pull/339#discussion_r327989630 je to tam takto pouzite vo vsetkych metodach tak to potom zmenim naraz